config/config.go

```diff
@@ -14,14 +14,12 @@ var Clock TimestampGenerator = RealTimestampGenerator{}
 // Path to Mist's "Livepeer" process that we shell out to for the transcoding
 const PathMistProcLivepeer = "./MistProcLivepeer"
 
+// Port that the local Broadcaster runs on
+const BroadcasterPort = 8935
```

Comment:
Hardcoding until we need to change it

Reply:
I have a change to read this from the transcode http request and parse into json to pass to MistProcLivepeer - left it out in my latest commit for now to reduce areas of debug. Will push it once we've worked through the critical bugs.